src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -448,68 +448,68 @@ export function getStatusContents(statusType, details = {}, name = {}) {
       );
       break;
     case STATUS_TYPES.ftr:
-      contents.title = 'Your appeal was closed';
+      contents.title = 'We closed your appeal';
       contents.description = (
-        <p>You did not take an action VA requested in order to continue your appeal. If this
-        information is incorrect, or if you want to reopen your appeal, please contact your Veteran
+        <p>You didn’t take action on something we requested that would have kept your appeal open. If this
+        information is incorrect, or if you want to reopen your appeal, please contact your Veterans
         Service Organization or representative for more information.</p>
       );
       break;
     case STATUS_TYPES.ramp:
-      contents.title = 'You opted in to the Rapid Appeals Modernization Program (RAMP)';
+      contents.title = 'You chose to take part in the Rapid Appeals Modernization Program (RAMP)';
       contents.description = (
         <div>
           <p>
             You chose to participate in the new supplemental claim or higher-level review lanes.
-            This does not mean that your appeal has been closed. If this information is incorrect,
-            please contact your Veteran Service Organization or representative as soon as possible.
+            This doesn’t mean that we’ve closed your appeal. If this information is incorrect,
+            please contact your Veterans Service Organization or representative as soon as possible.
           </p>
           <p>
-            At this time, Vets.gov is not able to provide information about appeals that are part
+            At this time, Vets.gov isn’t able to provide information about appeals that are part
             of RAMP.
           </p>
         </div>
       );
       break;
     case STATUS_TYPES.reconsideration:
-      contents.title = 'Your motion for reconsideration was denied';
+      contents.title = 'The Board denied your motion for reconsideration';
       contents.description = (
-        <p>The Board of Veterans’ Appeals declined to reopen your appeal. Please contact your
-        Veteran Service Organization or representative for more information.</p>
+        <p>The Board of Veterans’ Appeals decided not to reopen your appeal. Please contact your
+        Veterans Service Organization or representative for more information.</p>
       );
       break;
     case STATUS_TYPES.death: {
       const { first, middle, last } = name;
       const nameString = `${first || ''} ${middle || ''} ${last || ''}`;
-      contents.title = 'The appeal was closed';
+      contents.title = 'The Board closed this appeal';
       contents.description = (
-        <p>VA records indicate that {_.startCase(_.toLower(nameString))} is deceased, so this appeal has been closed. If
-        this information is incorrect, please contact your Veteran Service Organization or
+        <p>VA records indicate that {_.startCase(_.toLower(nameString))} is deceased, so the Board has closed this appeal. If
```

Comment:
Change to

VA records indicate that {_.startCase(_.toLower(nameString))} is deceased, so this appeal has been closed. If this information is incorrect, please contact your Veterans Service Organization or ...